.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,99 @@
+name: Bug Report
+description: File a bug report.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "**Required Info:**"
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: regression
+    attributes:
+      label: Regression
+      description: Is the reported bug a regression? If so, what is the last version of ROS where it still worked fine?
+  - type: input
+    id: os
+    attributes:
+      label: "Operating System:"
+      description: |
+        Please try to be specific.
+        For Linux, please use the command `uname -a` from a terminal and copy paste its output here.
+        For Windows, open a terminal (Win key + R and type `cmd`), type the command `ver` and press enter.
+        Then copy paste the output here.
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: "ROS version or commit hash:"
+      description: |
+        **Examples:**  *humble*, *jazzy*, ...
+    validations:
+      required: true
+  - type: input
+    id: rmw
+    attributes:
+      label: "RMW implementation (if applicable):"

```

Comment:
make this field as applicable since not all repositories do not require rmw implementation.

Reply:
Considering the frequency of DDS tuning issues we way also want to ask for DDS settings.

Reply:
that makes sense. i will add the optional textarea for this.